gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sat Nov 17 02:09:23 PST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```

Comment:
Can you please keep this as all?

Reply:
you mean the order of lines in the file ? Will leave it as is

Reply:
ok